docs/source/platform/schema-registry.md

```diff
@@ -1,102 +1,102 @@
 ---
-title: Registering your schema
-description: How to publish your schema to the Apollo registry
+title: Track schema change history
+description: Connect to the Apollo schema registry and track changes in your schema over time
 ---
 
-A schema is the center point of all GraphQL applications. It powers incredible development tools, is an always up to date set of documentation, and creates an enforcable contract between clients and servers through validation. Since this is such a center point of how teams work on GraphQL, the Apollo GraphQL Platform provides a free schema registry for teams of all sizes to use. In storing a schema in the registry, teams can share an always up to date picture of their data model into every facet of their workflow.
+The [GraphQL schema](https://www.apollographql.com/docs/tutorial/schema.html) is the center point of all GraphQL applications. It creates an enforceable contract between clients and servers, it is its own self-updating API documentation, and it provides powerful development workflows thanks to its type safety. As GraphQL scales in an organization and as GraphQL schemas grow, it becomes really important to have tools to help your team safely evolve your schema over time. At scale, you might have hundreds or thousands of clients in the world making queries that rely on fields, types, and arguments in your schema that are impossible to track without tooling.
 
-<h2 id="benefits">Benefits of publishing a schema</h2>
+At Apollo, we're driven to help teams be successful with GraphQL both at the initial adoption phase and as they scale GraphQL across their organization. As such, the [Apollo Platform](/docs/intro/platform.html) provides a free schema registration service for teams of all sizes to use. Much like version control for your code, the Schema Registry allows you to evolve your API over time while keeping a record of when each change was made and helping you catch breaking changes before they're every deployed.
 
-The Apollo schema registry provides a number of benefits to help teams collaborate and ship high quality software faster. A few of these benefits are:
+<h4 id="benefits">Benefits of registering your schema</h4>
```

Comment:
I'm not sure I would call these benefits, since we've already talked about the benefits above. Maybe developer tooling or workflows? I would also move this section to the bottom, make it an h2 or h3, and go straight into setup.